proposals/src/ledger-replication-to-implement.md

```diff
@@ -37,3 +43,100 @@ transacation proves the validator incorrectly validated a fake storage proof.
 The replicator is rewarded and the validator's staking balance is slashed or
 frozen.
 
+### Storage proof contract logic
+
+Each replicator and validator will have their own storage account. The validator's
+account would be separate from their gossip id similiar to their vote account.
```

Comment:
This has some maintenance problems, we need to store this keypair somewhere :( On the first pass I can just `keypair::new()` everytime, but aside from that they'll need to pass in a keypair to the validator somehow (like with vote accounts - but that thing is another monster). No way we can just reuse the validator's primary keypair right ?

Reply:
The primary keypair would have to be an account where account.owner == storage_program, so  I'm not sure if that's possible or *should* be the case for the validator's primary keypair.